### PR TITLE
feat/Customer Logout feature: navbar, actions, reducers and saga added

### PR DESCRIPTION
**Pull request by hemal-digiflux**

### CODE REVIEW

File: src/redux/saga/server.saga.ts [hunk 1]
```diff
@@ -27,12 +33,28 @@ export function* loginUser(action: userLoginActionInterface) {
     }
 }
 
+export function* logoutUser(action: userLogoutActionInterface) {
+    yield put(setLoadingTrue());
+    try {
+        yield put(push(CONFIG.routes.home));
```
Comment (Diego-Romero) [inline src/redux/saga/server.saga.ts]: This is good mate, but we are missing one small thing which is that there needs to be a request to the stub server which actually handles the logout procedures - in this case the cookie should be removed from the client via the server. I will update the stub server for you, you just need to trigger the request for /user/logout

Comment (hemal-digiflux) [reply src/redux/saga/server.saga.ts]: Changed

Comment (Diego-Romero) [reply src/redux/saga/server.saga.ts]: Thank you, just noticed that are only missing the Saga tests for this, would you mind including them please? They should be quite small.

**Comment by Diego-Romero** [comment]
Currently user logout saga also needs a test - left it for you as exercise, also the API functions need a test as well, that one should be dead easy. 👍 we seem to be on track mate!

File: src/redux/api.ts [hunk 2]
```diff
@@ -1,28 +1,51 @@
...
+        credentials,
+        body: JSON.stringify({ email, password }),
+    });
+    return await response.json();
 }
 
-// Sets cookie for the customer user
-export async function apiLoginCustomerUser(action: userLoginActionInterface) {
-    const res = await fetch(`${API_URL}user/login`, {
+export async function apiGetUserMe({ email, password }: UserLoginActionInterface): Promise<CustomerLoginResponse> {
```
Comment (Diego-Romero) [inline src/redux/api.ts]: remove as not being used

**Pull request merged**